tests/containers/agencyV2/AgencyContainerV2-test.jsx

```diff
@@ -76,3 +57,22 @@ test('an API request is made for the agency code in the URL', () => {
     });
 });
 
+// TODO - Commenting this out since it is failing our tests in the travis run even though tests are green, the rejection is causing an issue
```

Comment:
Are we expecting this to work with Travis at some point in the future? If not, I think we should delete this instead of commenting it out.

Reply:
Given this is the only test we have checking for a failing API call, we certainly could remove it. I see a lot of benefit though in being able to mock failing api calls and why i just commented it out.

Reply:
Leaving as an example and reminder to look into it!